dbt/scripts/export_models.py

```diff
@@ -87,20 +88,23 @@ def main():
         help="SQL expression representing a WHERE clause to filter models",
     )
 
-    args = parser.parse_args()
-    target = args.target
-    select = args.select
-    selector = args.selector
-    rebuild = args.rebuild
-    where = args.where
+    return parser.parse_args()
 
+
+def export_models(
+    target: str = "dev",
+    select: typing.List[str] | None = None,
```

Comment:
**question (non-blocking):** Why not use the new built-ins?

```suggestion
    select: list[str] | None = None,
```

Reply:
Embarrassingly, it was just because I didn't know about the new bulitins as of 3.9 😅 Fixed in 785ee5437f1b8ae616a30db1971dcd5535997de2 so that I could also remove the now-unused `typing` import.